src/app/_components/body-markdown.tsx

```
@@ -0,0 +1,25 @@
+export function BodyMarkdown({ content }: Props) {
+  const [html, setHtml] = useState<string>("");
+  useEffect(() => {
+    markdownToHtml(content).then((html) => {
+      setHtml(html);
+    });
+  }, [content]);
+
+  return (
+    <div
+      className={markdownStyles["markdown"]}
+      dangerouslySetInnerHTML={{ __html: html }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)